packages/sveltekit/src/mod.test.ts

```diff
@@ -0,0 +1,84 @@
+import { assertEquals } from "@std/assert";
+import { fedifyHook } from "./mod.ts";
+
+interface MockRequestEvent {
+  request: Request;
+}
+
+interface MockHookParams {
+  event: MockRequestEvent;
+  resolve: (event: MockRequestEvent) => Promise<Response>;
+}
+
+interface MockFederation<T> {
+  fetch(request: Request, options: any): Promise<Response>;
+}
+
+Deno.test("fedifyHook", async (t) => {
```

Comment:
In order to make this test runnable on Node.js and Bun, we should use [`node:test`](https://nodejs.org/api/test.html) and [`node:assert/strict`](https://nodejs.org/api/assert.html).